cicd/gitops/setting-up-argocd-instance.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="setting-up-argocd-instance"]
+= Setting up a new Argo CD instance
+include::_attributes/common-attributes.adoc[]
+:context: setting-up-argocd-instance
+
+toc::[]
+
+By default, the {gitops-title} installs an instance of Argo CD in the `openshift-gitops` namespace with additional permissions, which allows to manage certain cluster scoped resources.
+
+You can install and deploy a new Argo CD instance for managing cluster configurations or deploying applications. By default, any new instance will only have permissions to manage resources in the namespace where it is deployed.
+
+
+include::modules/gitops-argo-cd-installation.adoc[leveloffset=+1]
+
+include::modules/gitops-enable-replicas-for-argo-cd-server.adoc[leveloffset=+1]
+
+include::modules/gitops-deploy-resources-different-namespaces.adoc[leveloffset=+1]
+
+////
```

Comment:
It's unclear whether this Additional resources section is for the assembly or the module referenced by the include statement on line 18.

If it is for the assembly: 
- Use `== Additional resources` instead of `.Additional resources`.
- Specify an ID [as described here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies).

Reply:
Do you intend to keep the `////` comment block?